content/en/docs/ops/ambient/architecture/index.md

```diff
@@ -13,4 +13,77 @@ This page is under construction.
 
 ## Traffic routing
 
+In ambient mode, workloads can fall into 3 categories:
+* Uncaptured: this is a standard pod without any mesh features enabled
+* Captured: this is a pod that has traffic intercepted by `ztunnel`. Pods can be captured by setting the `istio.io/dataplane-mode=ambient` label on a namespace.
+* Waypoint enabled: this is a pod that is "Captured" *and* has a waypoint proxy deployed.
```

Comment:
```suggestion
* Waypoint enabled: this is a pod that is "Captured" *and* has a {{< gloss >}}waypoint proxy{{< /gloss >}} deployed.
```

Reply:
Noting the next line just uses waypoint, would it be better to have the single word in the glossary?